config/initializers/content_security_policy.rb

```diff
@@ -0,0 +1,28 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
I assume this file will be configured in a later commit.

Reply:
Like most other changes, this file got added by Rails upgrade scripts. I wasn't sure how to set this up, but since default setting seemed to work in rubyric-test, decided not to touch this for now.